recipes/root5/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "root5" %}
+{% set version = "5.34.36" %}
+{% set sha256 = "fc868e5f4905544c3f392cc9e895ef5571a08e48682e7fe173bd44c0ba0c7dcd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: root_v{{version}}.source.tar.gz
+  url: https://root.cern.ch/download/root_v{{version}}.source.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+   number: 0
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+ 
+ build:
+     - toolchain
+     - gcc  # [linux]
```

Comment:
`gcc` is provided on Linux. So this should not be needed unless it requires OpenMP or Fortran.

Reply:
Ok, I'll try and drop it. It was failing during my tests without that, but I'll look into it more.

Reply:
I see. Let's save it for the feedstock then.